### PR TITLE
Add Github Actions for CI

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.0', '1.4', 'nightly']
```
Comment (KristofferC) [inline .github/workflows/CI.yml:10]: Does GitHub action support `'1'` for the latest release?

Comment (thofma) [reply .github/workflows/CI.yml:10]: If I remember correctly it did not work when I tried. But let me try again.

Comment (thofma) [reply .github/workflows/CI.yml]: Yes, works.

Comment (thofma) [reply .github/workflows/CI.yml]: I changed it to use `['1.0', '1', 'nightly']`

**Comment by thofma** [comment]
Is there something else to address?
